src/Client.php

```diff
@@ -598,36 +612,33 @@ class Client
 
     public function __construct(array $params = [])
     {
-        $this->host = $params['host'] ?? null;
-        $this->username = $params['username'] ?? null;
-        $this->password = $params['password'] ?? null;
-        $this->integrator_key = $params['integrator_key'] ?? null;
+        $this->host = $params['host'] ?? config('docusign.host');
+        $this->auth_server = $params['auth_server'] ?? 'account-d.docusign.com';
+        $this->jwt_scope = $params['jwt_scope'] ?? "signature impersonation";
+        $this->integrator_key = $params['integrator_key'] ?? config('docusign.integrator_key');
+        $this->impersonated_user_id = $params['impersonated_user_id'] ?? config('docusign.impersonated_user_id');
+        $this->private_key = $params['private_key'] ?? config('docusign.private_key');
```

Comment:
We'd need to remove the usage of `config` and move that to the [Laravel-Docusign](https://github.com/Tucker-Eric/Laravel-Docusign) package due to `config()` not being defined globally here.
```suggestion
        $this->host = $params['host'] ?? null;
        $this->auth_server = $params['auth_server'] ?? 'account-d.docusign.com';
        $this->jwt_scope = $params['jwt_scope'] ?? "signature impersonation";
        $this->integrator_key = $params['integrator_key'] ?? null;
        $this->impersonated_user_id = $params['impersonated_user_id'] ?? null;
        $this->private_key = $params['private_key'] ?? null;
```

Reply:
Alright, let me make that change.